### ISSUE TITLE
Toggle lever not putting redstone signal into block

### ISSUE THREAD

**Issue by Vastitas** [Civcraft/JukeAlert#28]
Upon testing designs that theoretically work with redstone mechanics, we we're not able to make the redstone torches turn off via /jatogglelevers 1,  

Here is a gif of what occurs: http://gyazo.com/d16439aa7a86bdc8e523cbe90c370839

It is available to see on CivTest at x: -154 y: 70 z: 367

**Comment by ProgrammerDan** [comment]
The Jukebox itself is not becoming powered by the toggle event to my knowledge. Try putting a block again the toggle end, and connect the torches to turn off to that block:

                  [torch]
    [juke][switch][block]
                  [torch]

**Comment by rourke750** [comment]
I do not believe that it possible to use redstone torches.  You have to use levers.
